query-script.md

```diff
@@ -176,22 +176,25 @@ In this section we will create a data flow query by populating this template:
 import javascript
 import DataFlow::PathGraph
 
+predicate isCondition(Expr expr) { exists(ConditionGuardNode cgn | expr = cgn.getTest()) }
+
 class Config extends DataFlow::Configuration {
   Config() { this = "always true" }
 
   override predicate isSource(DataFlow::Node source) {
-    // fill in the body of this member predicate
+    exists(ObjectExpr obj |
+      obj = source.asExpr() and
+      obj.getAProperty().getName() = "verified"
+    )
   }
 
-  override predicate isSink(DataFlow::Node sink) {
-    // fill in the body of this member predicate
-    // you can use the isCondition() predicate we created earlier
-  }
+  override predicate isSink(DataFlow::Node sink) { isCondition(sink.asExpr()) }
 }
 
 from Config config, DataFlow::PathNode source, DataFlow::PathNode sink
-where            // fill in the where clause
+where config.hasFlowPath(source, sink)
 select sink, source, sink, "always true"
+
```

Comment:
Shouldn't this just stay a "template"? I would have left it as is?

Reply:
Yep - sorry copy and paste error. Good pick up 🙏🏼